src/effects/VST3/VST3Effect.cpp

```diff
@@ -395,11 +465,15 @@ bool VST3Effect::LoadSettings(
       } while(parms.GetNextEntry(key, index));
    }
 
+   VST3EffectSettings specificSettings;
+   FetchSettings(specificSettings);
+   GetSettings(settings) = specificSettings;
```

Comment:
Here and elsewhere, you might write just one line
`FetchSettings(GetSettings(settings))`

Reply:
But then, also, FetchSettings should clear the map before iterating the parameters and reinserting them.

Reply:
`FetchSettings` is called in all necessary places, good